include/variables.h

```diff
@@ -3311,9 +3311,9 @@ extern ShrinkWindowContext gShrinkWindowContext;
 extern ShrinkWindowContext* gShrinkWindowContextPtr;
 // extern UNK_TYPE4 D_801F6B50;
 // extern UNK_TYPE1 D_801F6B58;
-extern FaultAddrConvClient sKaleidoAreaFaultClient;
 extern void (*sKaleidoScopeUpdateFunc)(GlobalContext* globalCtx);
 extern void (*sKaleidoScopeDrawFunc)(GlobalContext* globalCtx);
+extern FaultAddrConvClient sKaleidoAreaFaultClient;
```

Comment:
Looks like this bss is already imported in their correct `c` files. Is there any problems if you just comment them out?

Reply:
Yeah it breaks bss all over.